cypress/integration/unit_tests/menu.spec.ts

```diff
@@ -28,7 +28,7 @@ describe('Menu testing', () => {
   });
 
   it('Verify Welcome Screen without Namespaces', () => {
-    cy.clickEpinioMenu('Namespaces');
+    cy.clickEpinioMenu('Namespacess');
```

Comment:
typo. 
I forced this error to check how the implementation worked.
I will revert this.